gnosis/safe-react
# fix: bytes params are truncated

## What it solves
Resolves #3592

## How this PR fixes it
Truncates Smart Contracts methods `bytes` values

## How to test it
Open a transaction details, for instance:
`eth:0x84443F61efc60D10DA9F9a2398980CD5748394BB/transactions/0xb176b051059e2fea8418e4206cca7e7e4cb99b52c18031a27660914d39991592`
- Observe that params of type `bytes` are now truncated

## Screenshots
<img width="654" alt="Screen Shot 2022-03-01 at 00 18 40" src="https://user-images.githubusercontent.com/32431609/156080798-17055281-dbbc-48fe-9c6a-92fff558bedd.png">
